tests/Meilisearch.Tests/KeyTests.cs

```diff
@@ -29,7 +30,6 @@ public async Task GetAllKeys()
         {
             var keyResponse = await this.client.GetKeysAsync();
             var keys = keyResponse.Results;
-
```

Comment:
Unintentional change? 🪀 

I like these spaces they help reading the structure of the test, do you know the **Four Phase-Test** structure? 
Knowing that I suggest you to revert this :D

Some refs:

- http://blog.plataformatec.com.br/2014/04/improve-your-test-readability-using-the-xunit-structure 
- http://xunitpatterns.com/Four%20Phase%20Test.html

Reply:
I wasn't aware of the pattern, but I have read the links and I can understand why the space is there, I've also reverted.